sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -400,4 +428,56 @@ export class ApplicationOfferingChangeRequestService {
       return applicationOfferingChangeRequest;
     });
   }
+
+  /** Validates student authorization for the provided application offering change request.
+   * @param applicationOfferingChangeRequestId application offering change request id.
+   * @param studentId student id for the authorization.
+   * @returns true if the application offering change request belongs to the student, otherwise false.
+   */
+  async validateStudentForOfferingChangeRequest(
+    applicationOfferingChangeRequestId: number,
+    studentId: number,
+  ): Promise<boolean> {
+    const { application } = await this.applicationOfferingChangeRequestRepo
+      .createQueryBuilder("applicationOfferingChangeRequest")
+      .select(["applicationOfferingChangeRequest.id", "application.id"])
+      .innerJoin("applicationOfferingChangeRequest.application", "application")
+      .innerJoin("application.student", "student")
+      .where(
+        "applicationOfferingChangeRequest.id = :applicationOfferingChangeRequestId",
+        {
+          applicationOfferingChangeRequestId,
+        },
+      )
+      .andWhere("student.id = :studentId", { studentId })
+      .getOne();
+
+    if (application) return true;
```

Comment:
And pls use brackets while using the `if` consditions

Reply:
@ann-aot  How about `return !!application;`?

Reply:
I believe, i already gave thumbs up to @dheepak-aot  comment, as I agree with him